### PR TITLE
Implement `Promise.any()`

### PR DESCRIPTION
**Pull request by retyui**
Issue: https://github.com/then/promise/issues/86

`Promise.any()` :
- Specification: https://tc39.es/ecma262/#sec-promise.any
- Docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any
- TS: https://github.com/microsoft/TypeScript/blob/main/lib/lib.es2021.promise.d.ts

### CODE REVIEW

File: src/es6-extensions.js [hunk 1]
```diff
@@ -140,3 +140,47 @@ Promise.race = function (values) {
...
+
+function getAggregateError(errors){
+  if(typeof AggregateError === 'function'){
+    return new AggregateError(errors,'All promises were rejected');
+  }
+
+  var error = new Error('All promises were rejected');
+
+  error.name = 'AggregateError';
+  error.errors = errors;
```
Comment (retyui) [inline src/es6-extensions.js:152]: I wandering is it correct way of polyfilling this `AggregateError`

Comment (ForbesLindesay) [reply src/es6-extensions.js:152]: I think this should be "good enough" as a polyfill.

**Review by ForbesLindesay — APPROVED** [review]

**Comment by ForbesLindesay** [comment]
I've just released this as v8.3.0

**Comment by retyui** [comment]
Thanks for quick review!

**Pull request merged**